src/pages/Chat/steps/AddressInput.tsx

```diff
@@ -116,19 +118,25 @@ export const AddressInput: React.SFC<AddressInputProps & Focusable> = ({
                     <TranslationsConsumer textKey="CHAT_INPUT_ADDRESS_POSTAL_NUMBER_PLACEHOLDER">
                       {(placeholder) => (
                         <UserTextInput
-                          type="number"
+                          type="text"
```

Comment:
how about we use `type="tel"` here? Then we will get the nice number keyboard despite allowing other characters.

I know it's non-semantic and a bit hacky though 🙁

Reply:
u sure? i don't think it does :/

Reply:
it's possible that I'm just imagining it 😞

Reply:
```suggestion
                          type="text"
                          inputMode="numeric"
```

Reply:
We'll save this for later.